backends/ref/ceed-ref-tensor.c

```diff
@@ -36,11 +36,6 @@ static int CeedTensorContractApply_Ref(CeedTensorContract contract, CeedInt A, C
   return CEED_ERROR_SUCCESS;
 }
 
-//------------------------------------------------------------------------------
```

Comment:
This is here intentionally - this isn't needed for functionality but the `ref` backend is the "example" other people can use to make their own backend, so I kept this in as in example.